content/docs/0.14.x/reference/files/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Files
+description: Definition of SLIs/SLOs of quality gate and how to customize them.
```

Comment:
```suggestion
description: Reference pages for Keptn files.
```
The sli/slo pages are only the beginning here.  This section will soon include pages for many other files.